src/Sulu/Bundle/ContactBundle/Resources/config/list-builder/Contact.xml

```diff
@@ -67,28 +74,47 @@
             </orm:joins>
         </property>
 
-        <property name="city" translation="contact.address.city" disabled="false" default="true">
+        <identity-property name="accountId" display="never" filter-type="auto-complete"
```

Comment:
because we can only filter for the id here and not for the name (the field-name is different). one is only used for the list and one only for filter